js/renovation/ui/scroll_view/utils/get_element_padding.ts

```diff
@@ -1,11 +1,11 @@
-import getElementComputedStyle from '../../../utils/get_computed_style';
+import { getElementComputedStyle } from '../../../utils/get_computed_style';
 import { toNumber } from '../../../utils/type_conversion';
 
 export function getElementStyle(
   name: keyof CSSStyleDeclaration, element: Element | null | undefined,
 ): number {
-  const computedStyle = getElementComputedStyle(element) ?? {};
-  return toNumber(computedStyle[name]);
+  const computedStyle = getElementComputedStyle(element as Element) ?? {};
```

Comment:
No.
It should be 
```
export function getElementPaddingBottom(element: Element): number {
  return toNumber(getElementComputedStyle(element).paddingBottom);
}
```
The 'getElementStyle' function is imported nowhere and we can delete it.

Reply:
Let's extract this change to a new PR with change to windowgetComputedStyle direct call.